school-sevice/src/test/java/liga/school/sevice/controller/SchoolControllerIntegrationTest.java

```diff
@@ -0,0 +1,306 @@
+package liga.school.sevice.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import liga.school.sevice.service.StudentService;
+import liga.school.sevice.transport.dto.SchoolCreateDto;
+import liga.school.sevice.transport.dto.SchoolOutComeDto;
+import liga.school.sevice.transport.dto.SchoolUpdateDto;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Iterator;
+
+import static org.mockito.ArgumentMatchers.anySet;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@Transactional
+public class SchoolControllerIntegrationTest {
+
+    @MockBean
+    private StudentService studentFeignService;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    @Sql("/scripts/initSchools.sql")
+    public void testGetStudents() throws Exception {
+        HashSet<String> studentIds = new HashSet<>(Arrays.asList("1", "2", "3"));
+        SchoolOutComeDto schoolOutComeDto = SchoolOutComeDto.builder()
```

Comment:
Why you create DTO? just read school from DB using repository
don't forget for similar places